FSJ_django20_project/FSJ/migrations/0003_auto_20180224_2027.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.0.2 on 2018-02-25 03:27
```

Comment:
Same question as with other migration file.

Reply:
Sorry, this one's my fault, didn't realize it had automatically been added